src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs

```diff
@@ -664,15 +664,15 @@ internal void AddDictionaryClaims(IDictionary<string, object> claimsCollection)
                 this[kvp.Key] = kvp.Value;
         }
 
-        internal static string GetClaimValueType(object value)
+        private static string GetClaimValueType(string claimType, object value)
```

Comment:
Is there a reason for adding new params to the begging? I think per guidelines, they're usually added to the end.

Reply:
Closer to the signature of the other methods.